app/src/main/java/skills/future/planer/ui/tasklist/Priorities.java

```diff
@@ -0,0 +1,15 @@
+package skills.future.planer.ui.tasklist;
+
```

Comment:
Przydał by się komentarz do czego to jest

Reply:
@M-Szymczyk nie trzeba, zajmij się czymś pożytecznym